package-lock.json

```diff
@@ -4,6 +4,23 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@ava/typescript": {
```

Comment:
Do we want to be including `package-lock.json`? We aren't in other packages.

Reply:
This is a separate project from Core and not in a monorepo/lerna situation, as such I can't think of a good reason *not* to.